### PR TITLE
Added GoalReacher and PostMailer to PostMutation

### PR DESCRIPTION
**Pull request by egonm12**

### CODE REVIEW

File: app/helpers/goal_helper.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module GoalHelper
+  def kudos_to_next_goal(team)
+    Goal.next(team).amount - team.active_kudos_meter.amount
```
Comment (StefanPahlplatz) [inline app/helpers/goal_helper.rb]: What happens when the last goal is reached?

File: app/mailers/post_mailer.rb [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
-    if receiver.name == ENV["COMPANY_USER"]
-      User.where.not(email: "").where.not(email: post.sender.email).where(deactivated_at: nil).each do |user|
-        suppress(Exception) { post_email(user, post).deliver_later if user.post_received_mail }
+    receivers.each do |receiver|
+      if receiver.email.present? && receiver.post_received_mail && receiver.deactivated_at.nil?
+        suppress(Exception) { post_email(receiver, post).deliver_now }
       end
     end
+
+    # TODO: Fix this method
```
Comment (StefanPahlplatz) [inline app/mailers/post_mailer.rb]: ?

File: app/helpers/goal_helper.rb (hunk 1 above)
Comment (egonm12) [reply app/helpers/goal_helper.rb]: In the post mail: `- if !Goal.next(@post.team).nil?`

File: app/mailers/post_mailer.rb (hunk 2 above)
Comment (egonm12) [reply app/mailers/post_mailer.rb]: Will get fixed when we can mention the 'company user'.

**Pull request merged**